lib/foreman_theme_satellite/branded_words.rb

```diff
@@ -7,11 +7,13 @@ module ForemanThemeSatellite
   # For now, modifiers are not supported by the generator.
   # Make sure the regexp works both with Ruby and JavaScript.
   # Entries are processed in order and first wins.
+  # Occurrences of "proxy" should be treated as a bug as it is unclear whether the author refers to a HTTP(S) proxy or a Capsule
   FOREMAN_BRAND = {
-    /%{proxy}/               => '%{proxy}',
-    /%{foreman}/             => '%{foreman}',
```

Comment:
I think these should also left intact: It's interpolation variables, and we don't want them branded.

Reply:
Re-added